### DISCUSSION TITLE
Cost of Expression.Compile()?

### DISCUSSION THREAD

**Discussion by pha3z** [sebastienros/jint#1548] [Ideas]
I noticed this library which claims to be a drop-in replacement for System.Linq.Expressions.  It claims it increases expression.Compile() 10x to 40x.

I was wondering what impact Expression performance has on jint Engine and whether replacing it with the faster library would make a noticeable improvement?

**Comment by lahma** [comment]
Jint doesn't use LINQ expressions in JS interpretation so I'd guess there would be no difference.
